pkg/helm/actions/install_chart_test.go

```diff
@@ -0,0 +1,33 @@
+package actions
+
```

Comment:
Can you sort the imports?

```go
// stdlib first
"io/ioutil"
"testing"
...
// third party 

// console 

```

Reply:
fixed https://github.com/openshift/console/pull/3826/commits/028957ef237401b6560830576595e84263996061